.github/copilot-instructions.md

```
@@ -0,0 +1,121 @@
+
+Use `bump-my-version` (NOT manual editing):
+
+```powershell
+poetry run bump-my-version bump patch  # 0.1.0 -> 0.1.1
+poetry run bump-my-version bump minor  # 0.1.0 -> 0.2.0
+poetry run bump-my-version bump major  # 0.1.0 -> 1.0.0
+```
+
+Updates version in: `pyproject.toml`, `src/drippy/__init__.py`, `CITATION.cff`, `docs/conf.py`
+
+**Dependencies**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)